Jamulus.pro

```diff
@@ -173,6 +173,17 @@ win32 {
     }
 
 } else:macx {
+    # Set minimum macOS deployment target
+    # For Qt 5: Support back to macOS 10.13 (High Sierra) for legacy builds on Intel
+    # For Qt 6: Support macOS 11 (Big Sur) and later for universal binaries
+    lessThan(QT_MAJOR_VERSION, 6) {
+        QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.13
+        message(Qt5 build: Setting macOS deployment target to 10.13)
+    } else {
+        QMAKE_MACOSX_DEPLOYMENT_TARGET = 11.0
+        message(Qt6 build: Setting macOS deployment target to 11.0)
+    }
+
```

Comment:
Is this needed?

Reply:
Apparently it ensures Xcode _knows_ which release we're targeting, otherwise it makes its own decision.

(I think there was an issue with it trying to use Qt6 on pre-macOS 11.  Or vice versa.  It was definitely Qt related, hence being a QMake directive.)

Reply:
Okay, that worked -- squashed.